ecms_base/ecms_base.install

```diff
@@ -2206,3 +2206,14 @@ function ecms_base_update_10203(array &$sandbox): void {
     $active_storage->write("{$configBlock}", $update_source->read("{$configBlock}"));
   }
 }
+
+/**
+ * Install the ecms_emergency_notification feature.
+ */
+function ecms_base_update_10204(array &$sandbox): void {
```

Comment:
This is updating existing sites, but how is this feature being installed on new sites?

Reply:
I added it to the ecms_base.info